pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/data/DemoBrowser.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.data
+
+import androidx.media3.common.MediaItem
+import androidx.media3.common.MediaMetadata
+
+/**
+ * Demo browser
+ *
+ * - root
+ *  - Playlist 1
+ *      - Playable items
+ *  - Playlist 2
+ *      - Playable items
+ *
+ * @property dataProvider
+ * @constructor Create empty Demo browser
+ */
+class DemoBrowser(private val dataProvider: DemoPlaylistProvider) {
+
+    private val mapMediaIdMediaItem = mutableMapOf<String, MediaItem>()
+    private val mapMediaIdToChildren = mutableMapOf<String, MutableList<MediaItem>>()
+
+    /**
+     * Root browsable item
+     */
+    val rootMediaItem: MediaItem by lazy {
+        MediaItem.Builder()
+            .setMediaId(DEMO_BROWSABLE_ROOT)
+            .setMediaMetadata(
+                MediaMetadata.Builder()
+                    .setFolderType(MediaMetadata.FOLDER_TYPE_MIXED)
+                    .setIsPlayable(false)
+                    .build()
+            )
+            .build()
+    }
+
+    init {
+        val rootList = mapMediaIdToChildren[DEMO_BROWSABLE_ROOT] ?: mutableListOf()
+        val listPlaylist = dataProvider.loadDemoItemFromAssets("streams.json")
+        for (playlist in listPlaylist) {
+            rootList += createPlaylistItem(playlist)
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SyntheticAccessor: Access to `private` method `createPlaylistItem` of class `Companion` requires synthetic accessor